File: sortInts.py
import sys

def sortIntsInFile (filename): 
    try: 
        #stores numbers from file in content
        content = [int(line.strip()) for line in open(filename, 'r')]
        content.sort()
        for num in content:
            print num
    except IOError:
        print("There is no file named " + filename);

if __name__ == '__main__':
    if len(sys.argv) != 2:
        print("Usage: python sortInts <intFileName>")
    else: 
        sortIntsInFile(sys.argv[1])

File: sortInts.java
import java.io.*; // Needed to use File and FileNotFoundException classes
import java.util.*; // Needed to use Scanner, ArrayList, and Collections classes

public class sortInts {

  public static void sortIntsInFile(String filename) {
    try {
      Scanner reader = new Scanner(new File(filename));
      ArrayList<Integer> numbers = new ArrayList<Integer>();
      while(reader.hasNextLine()){
        numbers.add(Integer.parseInt(reader.nextLine()));
      }
      Collections.sort(numbers);
      //System.out.println(numbers); //shows array before sorting
      for(int num: numbers){
        System.out.println(num);
      }
    } catch (FileNotFoundException e) {
      System.out.println("There is no file named " + filename);
    }
  }

  public static void main (String[] args) {
    if (args.length != 1) { 
      System.out.println("Usage: java sortInts <intFileName>");
    } else {
      sortIntsInFile(args[0]);
    }
  }
}